client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -22,14 +22,14 @@ interface Props {
 }
 
 export default function VerticalSelect( { inputClass, forceHideSuggestions, onClick }: Props ) {
-	const popular = [
-		NO__( 'Travel Agency' ),
-		NO__( 'Digital Marketing' ),
-		NO__( 'Cameras & Photography' ),
-		NO__( 'Website Designer' ),
-		NO__( 'Restaurants' ),
-		NO__( 'Fashion Designer' ),
-		NO__( 'Real Estate Agent' ),
```

Comment:
These are stubs to later replace by `__()` translate calls (once our strings have consolidated and ready for translation -- we're stubbing them out now in order not to cause extra work for translators). Why are we dropping them here?

Reply:
I've changed the array from a list of popular vertical labels, to a list of popular vertical "slugs". So it's more like a list of ids now, the "slugs" just happen to match the English label.
The labels presented in the UI were already coming from the API.

Reply:
I'm also curious about the i18n removal here. It's not removed form the static lists used in signup.

At this point, we might as well hard code the id and English translated text right here 😕 

This is a sore point we'd like to improve on, but for now I think it makes sense to stay close to changes in signup and only correct the text.

Reply:
The difference between the popular lists on `/gutenboarding` and `/start` is that the site-topic step doesn't wait for the verticals API response before displaying. It uses the hardcoded list.

I'm happy to switch back to `__()` though. We'll still need to change [the logic that filters for popular verticals](https://github.com/Automattic/wp-calypso/blob/1a9fe203710351d36869b39500c7da9d36e62a9f/client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx#L102). It needs to do an exact match, otherwise the `"Restaurant"` in `popular` will match with `"African Restaurant"` in the API response.